From adf3b6595bd71656f028c72f3c441442d9fb754b Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 11 Aug 2021 22:22:25 +0200
Subject: [PATCH 097/104] Revert "block/mq-deadline: remove redundant
 assignment of variable ret"

This reverts commit cb29bee87de410543254c339b2875ef6e48cf4de.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 block/mq-deadline-main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/mq-deadline-main.c b/block/mq-deadline-main.c
index d005cce73..edef13d56 100644
--- a/block/mq-deadline-main.c
+++ b/block/mq-deadline-main.c
@@ -639,6 +639,7 @@ static int dd_init_sched(struct request_queue *q, struct elevator_type *e)
 	if (ret)
 		goto free_stats;
 
+	ret = 0;
 	q->elevator = eq;
 	return 0;
 
-- 
2.33.0

